services/crypto_create.proto

```diff
@@ -30,147 +37,184 @@ import "basic_types.proto";
+     * early design dead-end.<br/>
+     * An amount below which record stream records would not be created for
+     * a transaction that increases this account balance.
      */
-    uint64 receiveRecordThreshold = 7 [deprecated=true];
+    uint64 receiveRecordThreshold = 7 [deprecated = true];
 
     /**
-     * If true, this account's key must sign any transaction depositing into this account (in
-     * addition to all withdrawals)
+     * A flag indicating the account holder must authorize all incoming
+     * token transfers.
+     * <p>
+     * If this flag is set then any transaction that would result in adding
+     * hbar or other tokens to this account balance MUST be signed by the
+     * identifying key of this account (that is, the `key` field).<br/>
+     * If this flag is set, then the account key (`key` field) MUST sign
+     * this create transaction, in addition to the transaction payer.
      */
     bool receiverSigRequired = 8;
 
     /**
-     * The account is charged to extend its expiration date every this many seconds. If it doesn't
-     * have enough balance, it extends as long as possible. If it is empty when it expires, then it
-     * is deleted.
+     * The duration between account automatic renewals.<br/>
+     * All entities in state may be charged "rent" occasionally (typically
+     * every 90 days) to prevent unnecessary growth of the ledger. This value
+     * sets the interval between such events for this account.
+     * <p>
+     * If the account balance (in HBAR) is insufficient to pay the full renewal
+     * fee, the entire HBAR balance SHALL be consumed and the expiration for
+     * the account SHALL be extended as far as the available balance can
+     * support.<br/>
+     * If the account HBAR balance is `0` when the account must be renewed, then
+     * the account SHALL be deleted, and subsequently removed from state.
      */
     Duration autoRenewPeriod = 9;
 
     /**
      * The shard in which this account is created
+     * <p>
+     * Currently, this MUST be `0`.<br/>
+     * If the desired shard is `0`, this SHOULD NOT be set.
      */
     ShardID shardID = 10;
 
     /**
-     * The realm in which this account is created (leave this null to create a new realm)
+     * The realm in which this account is created.
+     * <p>
+     * The shard number for this realm MUST match the value in `shardID`.<br/>
+     * Currently, this MUST be `0` for both fields.<br/>
+     * If the desired realm is `0.0`, this SHOULD NOT be set.
      */
     RealmID realmID = 11;
 
     /**
      * If realmID is null, then this the admin key for the new realm that will be created
+     * <blockquote>Should this be deprecated?<blockquote>
+     * The "create realm" was never enabled, and
+     * probably should not be possible on account creation.
+     * </blockquote></blockquote>
+     * Added deprecated tag 2024-05 based on above.
```

Comment:
Is this still an open question? Not sure if this is intended to actually be in the file, or if it's a discussion starter

Reply:
Thanks, replaced with deprecation reason.